kolibri/core/assets/src/views/ExamReport/TriesOverview.vue

```diff
@@ -115,30 +115,25 @@
           return 0.0;
         }
       },
+      // Returns the time spent on the best attempt or null if there are no attempts
       bestTimeSpent() {
-        const bestScoreAttempt = this.pastTries.find(t => t.correct === this.maxQuestionsCorrect);
-        if (!bestScoreAttempt) {
-          return null;
-        }
-        return bestScoreAttempt.time_spent;
+        return this.pastTries.length
+          ? this.pastTries.find(t => t.correct === this.maxQuestionsCorrect).time_spent
+          : null;
       },
+      // Returns the number of questions correct in the best attempt or 0 if there are no attempts
       maxQuestionsCorrect() {
-        return this.pastTries.length ? Math.max(...this.pastTries.map(t => t.correct)) : null;
+        return this.pastTries.length ? Math.max(...this.pastTries.map(t => t.correct)) : 0;
```

Comment:
Converted the `null` to a `0` here as anyhow the UI was showing the `null` as `0` in rendering.